ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -285,6 +292,23 @@ export const SimulationDetails: React.FC<SimulationDetailsProps> = ({
     transactionId,
   });
 
+  const { trackInlineAlertClicked } = useAlertMetrics();
+  const { getFieldAlerts } = useAlerts(transaction.id);
+  const fieldAlerts = getFieldAlerts(RowAlertKey.EstimatedApprovalChanges);
```

Comment:
Minor, should this be `RowAlertKey.EstimatedChangesStatic` as we're not aware of what the static rows are here?